JS/bloggeneration.js

```diff
@@ -0,0 +1,71 @@
+
+
+//blog page//
+
+
+
+const blog = document.createElement('div');
```

Comment:
I would consider wrapping this whole file in a for loop. You could eliminate some redundancy by having a list of dogs with attributes about the dogs within. 

For example:
```
dogs = [
'name': 'darcy',
'price: 120,
'img-source: './images/darcy.jpg'
]
```
for (let i = 0; i < dogs.length; i++) {
//create elements dynamically
}